src/components/Navigation/Pages/PageBrowse/PageBrowse.js

```diff
@@ -217,6 +217,7 @@ class PageBrowse extends SystemManagedList {
           <div className={classNames('section-body flex-grow', { loading: components.isFetching })}>
             <i className="fas fa-spinner fa-spin" />
             <ComponentList
+              multiSelectEnabled={false}
```

Comment:
I think this property should be moved to the SystemManagedList or UserManagedList components.

If it's a System Managed list, the `multiSelectEnabled` should be false by default.
Instead, if we are in a User Managed List, that it should be true.